index.html

```diff
@@ -300,6 +300,15 @@ <h2 style="margin-top:-20px;">Getting Involved: Hardware WG</h2>
 			<li>Mission is to reduce barriers to entry for using and developing hardware libraries in Node.js.</li>
 		</ul>
 	</div></section>
+	
+	<section class="slide"><div>
+		<h2 style="margin-top:-20px;">Getting Involved: Tracing WG</h2>
+		<ul>
+			<li>Mission is to provide a portable, high performance tracing API for Node.js.</li>
```

Comment:
Not sure a specific tracing API is exactly the goal. It's more tooling, and the hooks required to make the tools work.